.travis.yml

```diff
@@ -20,11 +20,16 @@ env:
   - LARAVEL=5.3.* TESTBENCH=3.3.* PHPUNIT=5.7.*
   - LARAVEL=5.4.* TESTBENCH=3.4.* PHPUNIT=5.7.*
   - LARAVEL=5.5.* TESTBENCH=3.5.* PHPUNIT=~6.0
+  - LARAVEL=5.6.* TESTBENCH=3.6.* PHPUNIT=~7.0
```

Comment:
Are you forced to specify the PHPUnit version too? It should be automatically chosen by Composer...

Reply:
Yes, since composer.json specifies `^4.6.6` and the other 2 dependencies also so they need to be overwritten in the composer file which is what these do.